airflow/health_check_test.go

```diff
@@ -0,0 +1,108 @@
+package airflow
+
+import (
+	"context"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+	"time"
+)
+
+func TestCheckWebserverHealthSuccess(t *testing.T) {
+	// Create a mock server that returns a 200 status
+	server := httptest.NewServer(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
+	}))
+	defer server.Close()
+
+	err := checkWebserverHealth(server.URL, 5*time.Second)
+	if err != nil {
+		t.Errorf("Expected no error, got: %v", err)
+	}
```

Comment:
nit: you could ignore it if you follow the current style
```suggestion
	assert.NoError(t, err)
```

Reply:
Fixed the full test suite to use the existing patterns.